src/audio/codec_adapter/codec_adapter.c

```diff
@@ -477,6 +517,19 @@ int codec_adapter_copy(struct comp_dev *dev)
 	}
 	ca_copy_from_module_to_sink(&local_buff->stream, md->mpd.out_buff, md->mpd.produced);
 
+	/* copy all produced output samples */
+	i = 0;
+	list_for_item(blist, &mod->sink_buffer_list) {
+		struct comp_buffer *buffer = container_of(blist, struct comp_buffer, sink_list);
+
+		if (mod->output_buffers[i].size > 0) {
+			ca_copy_from_module_to_sink(&buffer->stream, mod->output_buffers[i].data,
+						    mod->output_buffers[i].size);
+			audio_stream_produce(&buffer->stream, mod->output_buffers[i].size);
+		}
+		i++;
+	}
```

Comment:
I find this commit a bit confusing - it adds a list of sink buffers but also keeps the current single sink buffer, right? Below there's a line:
```
audio_stream_produce(&local_buff->stream, md->mpd.produced);
```
Maybe that's removed in a later commit, let's see